spark/src/main/java/com/nflabs/zeppelin/spark/SparkSqlInterpreter.java

```diff
@@ -76,13 +76,20 @@ public void open() {
     this.maxResult = conf.getInt("ZEPPELIN_SPARK_MAX_RESULT",
         "zeppelin.spark.maxResult",
         Integer.parseInt(getProperty("zeppelin.spark.maxResult")));
+    openSparkInterpreter(true);
```

Comment:
ya ...just having unconditional p.open in original getSparkInterpreter is good enough. I added rest of code because p.open is strictly reqd only we are opening this interpreter. I can make it unconditional and update PR